packages/manager/apps/pci-workflow/src/pages/list/List.page.tsx

```diff
@@ -54,14 +52,22 @@ export default function ListingPage() {
   const hrefAdd = useHref(`./new`);
   const [searchField, setSearchField] = useState('');
   const filterPopoverRef = useRef(undefined);
-  const { pagination, setPagination } = useDataGrid();
+  const [searchQueries, setSearchQueries] = useState<string[]>([]);
+  const {
+    pagination,
+    setPagination,
+    sorting,
+    setSorting,
+  } = useDatagridSearchParams();
```

Comment:
if there are modal actions on the same page searchParams will be reset when the modal is opened (url is changing) causing the datagrid pagination to reset aswell

in that case it's preferable to use useDataGrid hook